components/proxy/conf/Caddyfile

```diff
@@ -0,0 +1,251 @@
+{
+	# disable automatic SSL certificate generation
+	auto_https off
+	# disable admin API server
+	admin off
+
+	# set default SNI for old clients
+	default_sni {$GITPOD_DOMAIN}
+
+	# debug
+
+	# configure plugin order
+	# https://caddyserver.com/docs/caddyfile/directives#directive-order
+	order gitpod.cors_origin         before header
+	order gitpod.workspace_download  before redir
+}
+
+(compression) {
+	encode zstd gzip
+}
+
+# configure headers to force HTTPS and enable more strict rules for the browser
+(security_headers) {
+	header {
+		# enable HSTS
+		Strict-Transport-Security  max-age=31536000
+		# disable clients from sniffing the media type
+		X-Content-Type-Options     nosniff
+		# Define valid parents that may embed a page
+		Content-Security-Policy    "frame-ancestors self https://*.{$GITPOD_DOMAIN} https://{$GITPOD_DOMAIN}"
+		# keep referrer data off of HTTP connections
+		Referrer-Policy            no-referrer-when-downgrade
+		# Enable cross-site filter (XSS) and tell browser to block detected attacks
+		X-XSS-Protection           "1; mode=block"
+
+		defer # delay changes
+	}
+}
+
+(enable_log) {
+	log {
+		output stdout
+		format filter {
+			wrap json
+			fields {
+				logger			delete
+				msg				delete
+				size			delete
+				status			delete
+				resp_headers	delete
+				request			delete
+			}
+		}
+	}
+}
+
+(remove_server_header) {
+	header {
+		-server
+		-x-powered-by
+	}
+}
+
+(ssl_configuration) {
+	tls /etc/caddy/certificates/tls.crt /etc/caddy/certificates/tls.key {
+		#ca_root   <pem_file>
+	}
+}
+
+(upstream_headers) {
+	header_up X-Real-IP {http.request.remote.host}
+}
+
+(upstream_connection) {
+	lb_try_duration 1s
+}
+
+(debug_headers) {
+	header X-Gitpod-Region {$GITPOD_INSTALLATION_LONGNAME}
+}
+
+# Kubernetes health-check
+:8003 {
+	respond /live 200
+	respond /ready 200
+}
+
+# TODO: is this being used?
+http://health*.{$GITPOD_DOMAIN} {
+	import remove_server_header
+
+	respond "alive"
+}
```

Comment:
not anymore. Leftover from the `cerc` days

Reply:
fixed